src/platform/packages/shared/kbn-scout/src/cli/config_discovery.ts

```diff
@@ -45,6 +45,9 @@ export const discoverPlaywrightConfigs: Command<void> = {
         : `Found Playwright config files in '${pluginsMap.size}' plugins`;
 
     if (pluginsMap.size > 0 && flagsReader.boolean('save')) {
+      if (!fs.existsSync(SCOUT_OUTPUT_ROOT)) {
+        fs.mkdirSync(SCOUT_OUTPUT_ROOT, { recursive: true });
+      }
       const scoutConfigsFilePath = resolve(SCOUT_OUTPUT_ROOT, 'scout_playwright_configs.json');
```

Comment:
I'm wondering if we should have an entry in scout-info for that file that we could use right away here, e.g. `SCOUT_PLAYWRIGHT_CONFIGS`. What do you think?

Reply:
we can't, fails at compiling :(